demos/python_demos/image_retrieval_demo/main.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+"""
+ Copyright (c) 2019 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+import sys
+import time
+from argparse import ArgumentParser, SUPPRESS
+
+import numpy as np
+
+from image_retrieval_demo.image_retrieval import ImageRetrieval
+from image_retrieval_demo.common import central_crop
+from image_retrieval_demo.visualizer import visualize
+from image_retrieval_demo.roi_detector_on_video import RoiDetectorOnVideo
+
+INPUT_SIZE = 224
+
+
+def build_argparser():
+    """ Returns argument parser. """
+
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument('-m', '--model',
+                      help='Required. Path to an .xml file with a trained model.',
+                      required=True, type=str)
+    args.add_argument('-v', '--video',
```

Comment:
* It would be more consistent with other demos if `-i` stood for `Path to a video file or numeric camera ID.` and [-lb, --labels](https://github.com/opencv/open_model_zoo/blob/develop/demos/python_demos/action_recognition/README.md#running) or [gallery](https://github.com/opencv/open_model_zoo/tree/develop/demos/smart_classroom_demo#running) for `Path to a folder with gallery images.`
* Can the demo accept image/images for that key?

Reply:
* fixed
* it is supposed to work with specific videos only. Samples of videos and gallery, as well as model will be publicly available a bit later.